packages/thirdweb/src/auth/verify-hash.ts

```diff
@@ -129,8 +129,7 @@ export async function verifyHash({
   try {
     const result = await eth_call(rpcRequest, verificationData);
     return hexToBool(result);
-  } catch (err) {
-    console.error("Error verifying ERC-6492 signature", err);
+  } catch {
```

Comment:
What was the actual issue here?

6492 should automatically use 1271 if the account is already deployed, the deployed check happens inside the universal validator contract.

We shouldn't need to force it manually ever